src/core/or/circuitmux.c

```diff
@@ -1335,3 +1335,20 @@ circuitmux_compare_muxes, (circuitmux_t *cmux_1, circuitmux_t *cmux_2))
   }
 }
 
+/**
+ * Touch the circuitmux so its policy can be updated at this point in time.
+ */
+void
+circuitmux_touch(circuitmux_t *cmux)
```

Comment:
Would a better name for "touch" be "update policy", "recalculate priority", "calculate_initial_priority", or something similar?